src/FirstSteps/RPG/Adventures/Village.cs

```diff
@@ -0,0 +1,44 @@
+﻿using FirstSteps.RPG.Heroes;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FirstSteps.RPG.Items;
+
+namespace FirstSteps.RPG.Adventures
+{
+    class Village
+    {
+        public void Enter(Hero hero)
+        {
+            if (hero is Human)
+            {
+                Console.WriteLine("You enter to the Village!");
+            }
+            else
+            {
+                Console.WriteLine("You can't enter to the village! Only human can enter!");
```

Comment:
With code like this, you can still enter the Village even if you are not a Human.